tool/release/createnotes/Main.kt

```diff
@@ -21,6 +21,8 @@
 
 package com.vaticle.dependencies.tool.release.createnotes
 
+import java.nio.file.Files
```

Comment:
Most of the changes in this file are small changes propagated from the change made in `Note.kt`.